frontend/ui/elements/screen_notification_menu_table.lua

```diff
@@ -76,5 +78,28 @@ This allows selecting which to show or hide.]]),
             end,
             separator = true,
         },
+        {
+            text = _("Show past notifications"),
+            help_text = _("Show the text of past Notifications."),
```

Comment:
```suggestion
            help_text = _("Show the text of past notifications."),
```

Seems a redundant though? I'd basically the same.